baremetal_network_provisioning/drivers/snmp_driver.py

```diff
@@ -35,7 +34,6 @@ class SNMPDriver(driver.PortProvisioningDriver):
 
     def set_isolation(self, port):
         """set_isolation ."""
-        LOG.info(_LI("set_isolation with port '%s' "), port)
```

Comment:
Since there is no log messages for other methods, i thought of removing this statements